src/pages/EditSplitBillPage.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import {withOnyx} from 'react-native-onyx';
+import compose from '../libs/compose';
+import CONST from '../CONST';
+import ROUTES from '../ROUTES';
+import Navigation from '../libs/Navigation/Navigation';
+import ONYXKEYS from '../ONYXKEYS';
+import reportPropTypes from './reportPropTypes';
+import transactionPropTypes from '../components/transactionPropTypes';
+import * as ReportActionsUtils from '../libs/ReportActionsUtils';
+import * as ReportUtils from '../libs/ReportUtils';
+import * as TransactionUtils from '../libs/TransactionUtils';
+import * as IOU from '../libs/actions/IOU';
+import * as CurrencyUtils from '../libs/CurrencyUtils';
+import FullPageNotFoundView from '../components/BlockingViews/FullPageNotFoundView';
+import EditRequestDescriptionPage from './EditRequestDescriptionPage';
+import EditRequestMerchantPage from './EditRequestMerchantPage';
+import EditRequestCreatedPage from './EditRequestCreatedPage';
+import EditRequestAmountPage from './EditRequestAmountPage';
+
+const propTypes = {
+    /** Route from navigation */
+    route: PropTypes.shape({
+        /** Params from the route */
+        params: PropTypes.shape({
+            /** Which field we are editing */
+            field: PropTypes.string,
+
+            /** reportID for the "transaction thread" */
+            threadReportID: PropTypes.string,
+        }),
+    }).isRequired,
+
+    /** The report object for the thread report */
+    report: reportPropTypes,
+
+    /** Used for retrieving the draft transaction of the split bill being edited */
+    draftSplitTransactions: PropTypes.shape(transactionPropTypes),
+
+    /** Session info for the currently logged in user. */
+    session: PropTypes.shape({
+        /** Currently logged in user email */
+        email: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    report: {},
+    session: {
+        email: null,
+    },
+    draftSplitTransactions: {},
+};
+
+function EditSplitBillPage({report, route, draftSplitTransactions}) {
+    const fieldToEdit = lodashGet(route, ['params', 'field'], '');
+    const reportAction = ReportActionsUtils.getReportAction(report.reportID, lodashGet(route, ['params', 'reportActionID'], ''));
+    const transaction = TransactionUtils.getLinkedTransaction(reportAction);
+
+    let draftSplitTransaction = draftSplitTransactions[`${ONYXKEYS.COLLECTION.DRAFT_SPLIT_TRANSACTION}${transaction.transactionID}`];
+    if (!draftSplitTransaction) {
+        IOU.setDraftSplitTransaction(transaction.transactionID);
+    }
```

Comment:
```suggestion
```

I don't think it's necessary to init the draft. The draft will be created once we make a call with `transactionChanges`